modules/rosa-unlinking-and-deleting-ocm-and-user-iam-roles.adoc

```diff
@@ -1,16 +1,41 @@
 // Module included in the following assemblies:
 //
 // * rosa_install_access_delete_clusters/rosa-sts-deleting-cluster.adoc
+// *rosa_hcp/rosa-hcp-deleting-cluster.adoc
+
+ifeval::["{context}" == "rosa-hcp-deleting-cluster"]
+:hcp:
+endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id="rosa-unlinking-and-deleting-ocm-and-user-iam-roles_{context}"]
 = Unlinking and deleting the {cluster-manager} and user IAM roles
 
-If you installed a {product-title} (ROSA) cluster by using {cluster-manager-first}, you created {cluster-manager} and user Identity and Access Management (IAM) roles and linked them to your Red Hat organization. After deleting your cluster, you can unlink and delete the roles by using the ROSA CLI (`rosa`).
+If you installed a 
```

Comment:
This is issue exists in the original content, but the _if_ is throwing me a little bit. For some reason, I start reading the sentence as a list, dragging the if along through the second clause.

Something like the following is a little easier for me to follow: "When you install a ROSA cluster..., you also create cluster manager and IAM roles that you link to your RH org. After you delete your cluster, you can unlink and delete...

Anyhow, the content is completely fine as written. I am just splitting hairs because you asked for a full critique. :)